packages/gatsby/src/services/calculate-dirty-queries.ts

```diff
@@ -0,0 +1,22 @@
+import { calcInitialDirtyQueryIds, groupQueryIds } from "../query"
+import { IBuildContext } from "./"
+
+export interface IGroupedQueryIds {
+  pageQueryIds: string[]
+  staticQueryIds: string[]
+}
+
+export async function calculateDirtyQueries({
+  store,
+  filesDirty,
+}: Partial<IBuildContext>): Promise<{
+  queryIds: IGroupedQueryIds
+}> {
+  const state = store?.getState()
+  if (filesDirty) {
+    // Do stuff
+  }
```

Comment:
Let's remove this from this PR and re-add when we have more things in place when we actually can "do stuff" here ;)